Assets/Prefabs/Player.prefab

```diff
@@ -317,7 +317,7 @@ MonoBehaviour:
   GroundDistance: 0.4
   GroundMask:
     serializedVersion: 2
-    m_Bits: 2147483647
+    m_Bits: 8
```

Comment:
The player prefab needed a reference to the terrain layer mask, which was missing